crates/json-log/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "jsparagus-json-log"
+version = "0.1.0"
+authors = ["The jsparagus Project Developers"]
+edition = "2018"
+license = "MIT/Apache-2.0"
+
+[dependencies]
+log = { version = "0.4.0", features = ["max_level_debug", "release_max_level_warn"] }
+serde_json = "1.0"
```

Comment:
If you can add the following:

```toml
[feature]
default = []
logging = ["log", "serde_json"]
```

as well as making these 2 dependencies optional, using `optional = true`